src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneModels/DataLakeStoreItem.cs

```diff
@@ -0,0 +1,101 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.DataLake.Store;
+
+namespace Microsoft.Azure.Commands.DataLakeStore.Models
+{
+    /// <summary>
+    /// The object that is used to manage permissions for files and folders. Moved the members of it's parent class to the current class. 
+    /// This is an exposed model to public so it should be independent of SDK models. We will never be able to remove members. In future if we have a different sdk then
+    /// we just need to map the SDK model to this model.
+    /// </summary>
+    public class DataLakeStoreItem
+    {
+        public DateTimeOffset LastWriteTime { get; }
+        public DateTimeOffset LastAccessTime { get; }
+
+        public DateTimeOffset? Expiration { get;}
+
+        public string Name { get; internal set; }
+
+        public string Path { get; internal set; }
+
+        public long? AccessTime { get; }
+        /// <summary>Gets the block size for the file.</summary>
+        public long? BlockSize { get; }
```

Comment:
@rahuldutta90 nit: would you mind adding an empty line between each property?

Reply:
Fixed.